stacktrace.go

```diff
@@ -199,25 +197,15 @@ func fileContext(filename string, line, context int) ([][]byte, int) {
 	return lines[start:end], idx
 }
 
-var trimPaths []string
-
-// Try to trim the GOROOT or GOPATH prefix off of a filename
+// trimPath tries to trim the GOROOT or GOPATH prefix off of a filename.
```

Comment:
I think this comment is now irrelevant, right? Since we don't look at these values at all.